tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/modules/myService/helloService.bal

```diff
@@ -0,0 +1,15 @@
+//import ballerina/http;
```

Comment:
Do we need this completely commented out file?

Reply:
The project won't compile as this module is having std-lib dependencies, so the plan is to migrate the before mentioned HTTP service based tests into the distribution repo. +1 to remove the related content until that. Fixed with 9e0ad3a